OWNERS.md

```diff
@@ -6,14 +6,26 @@ This page lists all active maintainers and their areas of expertise. This can be
 routing PRs, questions, etc. to the right place.
 
 * Alan Chiu ([buildbreaker](https://github.com/buildbreaker)) (achiu@lyft.com)
-  * Android
-* Matt Klein ([mattklein123](https://github.com/mattklein123)) (mklein@lyft.com)
-  * Community, code reviews, design reviews, etc.
+  * Android (kotlin/java) platform bindings.
 * Jose Nino ([junr03](https://github.com/junr03)) (jnino@lyft.com)
-  * Envoy mobile core
+  * Envoy mobile core.
 * Michael Rebello ([rebello95](https://github.com/rebello95)) (mrebello@lyft.com)
-  * iOS
+  * iOS (swift/objective-c) platform bindings.
 * Mike Schore ([goaway](https://github.com/goaway)) (mschore@lyft.com)
-  * Envoy mobile core
+  * Envoy mobile core.
+
+# Friends of Envoy Mobile
```

Comment:
@jingwei99 @crockeo @carloseltuerto lmk if you feel comfortable being listed here.

Reply:
That's fine with me - thanks.